v1.1/admin-ui-access-and-navigate.md

```diff
@@ -0,0 +1,103 @@
+
+<img src="{{ 'images/admin_ui_overview.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+### Time Series Graphs
+
+The Admin UI displays time series graphs of key metrics. Time series graphs are useful to visualize and monitor data trends. You can hover over each graph to see actual point-in-time values. 
+
+<img src="{{ 'images/admin_ui_hovering.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+#### Change time range
+
+You can change the time range by clicking on the time window.
+<img src="{{ 'images/admin_ui_time_range.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+{{site.data.alerts.callout_info}}The Admin UI shows time in UTC, even if you set a different time zone for your cluster. {{site.data.alerts.end}}
+
+#### View metrics for a single node
+
+By default, the time series panel displays the metrics for the entire cluster. To view the metrics for an individual node, select the node from the **Graph** drop-down list.
+<img src="{{ 'images/admin_ui_single_node.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+### Summary Panel
+<img src="{{ 'images/admin_ui_summary_panel.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:40%" />
+
+The Summary panel provides information about:
+
+Metric | Description
+--------|----
+Total Nodes | The total number of nodes in the cluster.<br><br>You can further drill down into the nodes details by clicking on **View nodes list**.<br><br>Note: <a href='admin-ui-access-and-navigate.html#decommissioned-nodes'>Decommissioned nodes</a> are not included in the Total Nodes count displayed on the Summary panel.
+Dead Nodes | The number of [dead nodes](admin-ui-access-and-navigate.html#dead-nodes) in the cluster.  
+Capacity Used | The storage capacity used as a percentage of total storage capacity allocated across all nodes.<br><br>Note: If you are running multiple nodes on a single machine (not recommended), then the value of the Capacity Used metric displayed on the Summary panel may be incorrect. This is because when multiple nodes are running on a single machine, the machine's hard disk is treated as separate stores for each node, while in reality, only one hard disk is used for all nodes. The Capacity Used is then displayed as the hard disk capacity used multiplied by the number of nodes on the machine. 
+Unavailable Ranges | The number of unavailable ranges in the cluster. A non-zero number indicates an unstable cluster. 
+Queries per second | The number of SQL queries executed per second. 
+P50 Latency | The 50th percentile of service latency. The service latency is calculated as the time between receiving a query and returning a response. This time does not include network latency between the node and client. 
```

Comment:
Let's replace the second and third sentences with this, which we use on another page:

> Service latency is calculated as the time between when the cluster receives a query and finishes executing the query. This time does not include returning results to the client.